app/core/crossref/contributors.tsx

```diff
@@ -1,23 +1,86 @@
 import { Element, Text } from "xast"
 
```

Comment:
This file is getting a little messy, i could also move these all to a separate "types" file, just so they're out of the way.
Btw I took most of these from the `xsd-to-xast` generator